build-scripts/bundle.js

```diff
@@ -53,14 +53,16 @@ module.exports.definedVars = ({ isProdBuild, latestBuild, defineOverlay }) => ({
   ...defineOverlay,
 });
 
-module.exports.htmlMinifierOptions = {
+const htmlMinifierOptions = {
   caseSensitive: true,
   collapseWhitespace: true,
   conservativeCollapse: true,
   decodeEntities: true,
   removeComments: true,
   removeRedundantAttributes: true,
-  minifyCSS: true,
+  minifyCSS: {
+    level: 0,
```

Comment:
What did it break?

Reply:
The fab is one of the first we noticed, it removes units and rules, which actually make the rules invalid

Reply:
I figured the defaults were conservative... I guess not.

Reply:
Here is the screenshot of the fab.

![image](https://user-images.githubusercontent.com/1444314/222169394-c411004d-80ec-4ae6-92cf-615e89f80ef4.png)

Reply:
Is this still removing whitespace?  The doc confusingly has it as a `format` option and then again under level 1 optimization:

>       removeWhitespace: true, // controls removing unused whitespace; defaults to `true`